model/src/main/kotlin/PackageCurationData.kt

```diff
@@ -116,6 +116,8 @@ data class PackageCurationData(
             isMetaDataOnly = isMetaDataOnly ?: basePkg.isMetaDataOnly
         )
 
-        return CuratedPackage(resultPackage, base.curations + PackageCurationResult(basePkg.diff(resultPackage), this))
+        val resultCurations = base.curations + PackageCurationResult(basePkg.diff(resultPackage), this)
```

Comment:
What about `appliedCurations`? `result` is so generic.

Reply:
...as in previous change: to distinguish between the outut (result) and the input (base).

Reply:
I've changed the approach a bit.
Does it resolve the issue?